WorkflowFunctionAdminBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -0,0 +1,46 @@
+{% block workflow_right_widget %}
+    {% spaceless %}
+        <div class="table-responsive">
+            <table class="table table-bordered table-striped workflow-right-widget">
```

Comment:
table is used by smartadmin and ticket : (2) Etq Ubo, je vois les droits de workflow ss la forme d'un tableau à double entrée.
So there are always alternatives. But this solution is the best for me. @itkg-ngilain